### ISSUE TITLE
Web page doesn't load, stratum server works fine.

### ISSUE THREAD

**Issue by VegArchie** [remcoros/public-pool-startos#9]
The entire server and web UI worked fine and then suddenly the web UI stopped working. Cannot load the page. Web browser says "server IP address could not be found." Miners are still connected as the stratum server is still working.

Running on Start OS server. Coincidentally, today, another installation of public pool, on a separate device, suddenly started a very similar issue. Web page won't load and says "bad gateway."

Any suggestions? Are the two related and is this a general public pool bug? Anyone else seeing this?

**Comment by remcoros** [comment]
Are the web ui's of other services on your server working? This looks like a problem with '.local' urls, and most likely related to MDNS (Bonjour on windows). see: https://docs.start9.com/0.3.5.x/user-manual/connecting-lan#windows-only

You can try restarting the Bonjour service on Windows.

I sometimes have this when my Windows PC resumes from sleep, a lot of the time, all my '.local' urls stop working until bonjour is restarted.

**Comment by VegArchie** [comment]
Using Linux and android. However at some point today it started working on my Linux machine (but not android). Then later in day, android started working. I have no idea what changed. I had tried rebooting earlier with no change. This became a problem and repaired itself on its own.

**Comment by remcoros** [comment]
> Using Linux and android. However at some point today it started working on my Linux machine (but not android). Then later in day, android started working. I have no idea what changed. I had tried rebooting earlier with no change. This became a problem and repaired itself on its own.

Yeah, it's all MDNS issues. It's a regular issue mentioned on Start9 forums and support channels. One of the reasons Start9 is moving away from '.local' urls and start using IP:port mappings in 0.3.6.

I'll close this, as it was unrelated to public-pool.
